src/reference/sectionA_definitions.inc

```diff
@@ -138,25 +138,24 @@ Non-specific information items
 
    #. A comment information item;
 
-   #. A namespace information item;
+   #. A namespace information item; or
 
    #. A character information item.
 
-2. An element information item in the :ref:`CellML namespace<specA_cellml_namespace>` MUST NOT contain any character information items, except for whitespace characters.
+2. An element information item in the :ref:`CellML namespace<specA_cellml_namespace>` MUST NOT contain any character information items, except for :ref:`whitespace characters<specA_whitespace_character>`.
 
 .. marker_cellml_information_sets_3
 
+.. _specA_semantic_equivalence:
+
 3. Two :ref:`CellML infosets<specA_cellml_infoset>` SHALL be deemed semantically equivalent if one can be transformed into the other by making zero or more of the following changes:
 
    #. Adding, removing, and/or modifying comment information items.
 
    #. Changing (inserting, removing, and/or modifying) one or more namespace information items, and/or modifying the prefix of one or more information items, without changing the namespace that any information item is in.
 
    #. The following paragraph applies to character information items which are the direct child of an element information item in the :ref:`CellML namespace<specA_cellml_namespace>`, or in the :ref:`MathML namespace<specA_mathml_namespace>`:
-
-      Inserting or removing character information items that consist entirely of :ref:`whitespace characters<specA_whitespace_character>`,
-      changing the number of whitespace characters in such an information item,
-      or changing the number of whitespace characters at the beginning or end of any character information item.
+      inserting or removing character information items that consist entirely of :ref:`whitespace characters<specA_whitespace_character>`, changing the number of whitespace characters in such an information item, or changing the number of whitespace characters at the beginning or end of any character information item.
```

Comment:
Curious why you did this? Also we've been using ; inside lists, no?

Reply:
Oh cause it's rendered that way, I suppose?

Reply:
to stop the page break in the latex rendering from breaking it before the "Inserting ..."

Reply:
I believe it was to avoid an unfortunate pagebreak in the PDF.

Reply:
Ouch. Don't change the content to fix the formatting :D :D :D

Reply:
I'll just pretend you didn't say that

Reply:
thats not nice. Can't we do something like the following in the right place instead?

```
 .. raw:: latex
 
      \newpage % hard pagebreak at exactly this position
```